### DISCUSSION TITLE
How to create new Bin. file after modding the settup files ?

### DISCUSSION THREAD

**Discussion by iceman00700** [BitMaker-hub/NerdMiner_v2#359] [Q&A]
Sorry in advance very new to all this.
i have downloaded the Nerdminer v2 files to my esp32-wroom and can not get it to work with my st7735 screen.
I can not see a list for pinouts for the wroom file .
so started looking at the setup files and found some sections to do with the screen i mentioned , i have edited them or so i think anyway but i am having issues generating new Bin files.

if anybody can help that would be amazing

**Comment by paelzer** [comment]
Hi

I didn't work with the bin files yet but compiling and flashing the code with VScode & Platformio is pretty easy.

I have uploaded a working quick & dirty draft to run the Nerdminer on an ESP32 connected to an ST7735 TFT (128x160px), find it here: https://github.com/paelzer/ESP32_ST7735_Nerdminer_V2. 

It's based on the LiligoT-QT so the screen is not fully occupied as LiligoT-QT has only 128x128px. Also no push buttons were connected.

Find the pinout I used in the bottom of platformio.ini file (env: ESP32_ST7735):

![image](https://github.com/BitMaker-hub/NerdMiner_v2/assets/16374477/3231588a-9446-4e01-a5a6-10dd9316b08c)


Use the env called: ESP32_ST7735 to compile from the dropdown in the bottom of VScode (red line) and the correct port (green line), compile and flash with the arrow button (blue line):

![image](https://github.com/BitMaker-hub/NerdMiner_v2/assets/16374477/cf1dc317-c6c2-4deb-a87a-8727043d2359)



![image](https://github.com/BitMaker-hub/NerdMiner_v2/assets/16374477/50e22afb-5e5c-48c9-82a6-618766fdcdfc) 
![image](https://github.com/BitMaker-hub/NerdMiner_v2/assets/16374477/56852578-9690-4f40-9768-efca08c96e24)

BR
Boris

**Reply by iceman00700** [reply]
Thanks for the help i will give it a go and if i manage to get it to full screen i will post it out .

Thank you again

**Reply by paelzer** [reply]
OK, sounds good. The images are c arrays so these need to be recreated for 128x160px. Not sure if somebody has the original files to edit them and then convert them into c arrays. Maybe it could be done in a different way, don't know. Don't have much experience in working with graphics.
